package.json

```diff
@@ -578,6 +587,11 @@
           "when": "view == confluent-resources && (viewItem == ccloud-kafka-cluster || viewItem == local-kafka-cluster)",
           "group": "topics"
         },
+        {
+          "command": "confluent.topics.openlatestschemas",
+          "when": "view == confluent-topics && viewItem =~ /.*-topic.*-with-schema.*/",
+          "group": "topics"
```

Comment:
Minor: I think we'll want to come up with a new `schema` flavored group here. As is, this gets grouped alongside the "Delete Topic" command (if available), whereas we could use something like `topic_schemas@1` and have it show up in its own group:
<img width="280" alt="image" src="https://github.com/user-attachments/assets/734d5510-64e0-4a75-8909-72d35cfa138f">

Reply:
Cool, will do.